quickstarts/flutter-web/config.yml

```diff
@@ -0,0 +1,49 @@
+
+
```

Comment:
Let's remove this white space

Reply:
Hi @brammerl ,
Removed white spaces. Could you review it and if need any changes let me know.

Thanks...